src/Umbraco.Core/Services/Implement/ContentService.cs

```diff
@@ -624,6 +647,15 @@ private IQuery<IContent> GetPagedDescendantQuery(string contentPath)
 
             return _documentRepository.GetPage(query, pageIndex, pageSize, out totalChildren, filter, ordering);
         }
+        private IEnumerable<IContent> GetPagedLocked(IQuery<IContent> query, long pageIndex, int pageSize,
```

Comment:
This code can be shared in one method which can accept a bool as to whether or not to include the count